logprep/processor/generic_adder/processor.py

```diff
@@ -46,8 +46,8 @@ def __init__(self, name: str, configuration: dict, logger: Logger):
         generic_rules_dirs = configuration.get("generic_rules")
         super().__init__(name, tree_config, logger)
         self.ps = ProcessorStats()
-        self._generic_tree = RuleTree(tree_config)
-        self._specific_tree = RuleTree(tree_config)
+        self._generic_tree = RuleTree(config_path=tree_config)
+        self._specific_tree = RuleTree(config_path=tree_config)
```

Comment:
```suggestion
```
as you can see [here ](https://github.com/fkie-cad/Logprep/blob/b695bb0e9872b9ea14f3ffa71a1c186e1991e677/logprep/processor/base/processor.py#L128)it is set in the base class and can be deleted

Reply:
ah even better, thanks!